src/week3/mar6/JavaMethodOverriding.java

```diff
@@ -61,3 +61,30 @@ public static void main(String[] args) {
 	}
 
 }
+/*
+I would like to get the below output as mentioned in the question:
+Generic Sports
+Each team has n players in Generic Sports
+Soccer
+Each team has n players in Soccer
+Volleyball
+Each team has 6 players in Volleyball
+Generic Sports
+Each team has 11 players in Generic Sports
+
+However, with the above code I'm getting below output which isn't what I expected:
+Generic Sports
+Each team has n players in Generic Sports
+Soccer
+Each team has n players in Soccer
+Volleyball
+Each team has 6 players in Volleyball
+Cricket
+Each team has 11 players in Cricket
+
+Please make changes to the above code to display the desried output.
+
+NOTE: You don't need to override every method if you're not willing to change the logic in the overridden method.
+Only override the particular method whose logic you wish to change in the child class.
+If you don't override a method in a child class, automatically it's parent class implementation for that method is called.
+*/
```

Comment:
Sure . i thought the question is wrong as in place of cricket there it was mistaken as generic... ill edit and commit again